expr.go

```diff
@@ -161,6 +161,19 @@ func (neq NotEq) ToSql() (sql string, args []interface{}, err error) {
 	return Eq(neq).toSQL(true)
 }
 
+// Between is syntactic sugar for use with BETWEEN methods.
+// Ex:
+//     .Where(Between{field: "id", left: 1, right: 5}) == "id between 1 and 5"
+type Between struct {
+	field string
+	left  interface{}
+	right interface{}
```

Comment:
These fields are all unexported, so you wouldn't be able to instantiate this struct.